apcd-cms/src/apps/submissions/urls.py

```diff
@@ -1,8 +1,12 @@
 from django.urls import path
 from apps.submissions.views import SubmissionsTable
 from apps.admin_regis_table.urls import RegistrationsTable
+from apps.exception.views import ExceptionFormView, ExceptionThresholdFormView, ExceptionOtherFormView
 
 app_name = 'submissions'
 urlpatterns = [
     path('list-submissions/', SubmissionsTable.as_view()),
-]
\ No newline at end of file
+    path('exception/', ExceptionFormView.as_view(), name="exception"),
+    path('threshold-exception/', ExceptionThresholdFormView.as_view(), name="threshold-exception"),
+    path('other-exception/', ExceptionOtherFormView.as_view(), name="other-exception")
+]
```

Comment:
I don't think the exception paths are needed here since you already included them through `exception/urls.py` file

Reply:
Ah good catch. So I tried to remove this from the submissions url but the the pages would not longer appear on that path. So instead, I removed the URL patterns from the exception urls.py since that is not longer needed. exceptions/urls.py did need at least one url pattern, so I left the main landing page /exception/ with name='index' and that seemed to work.